---
mass: 1
gra: 9.81
hov_percent: 0.72
full_thrust: 13.6
pose_solver: 1
ctrl_freq_max: 100
use_bodyrate_ctrl: false
max_manual_vel: 1
max_angle: 30
low_voltage: 13.2
leader: true
debug: true
perform_aerodynamics_compensation: false
rc_reverse:
  roll: false
  pitch: false
  yaw: false
  throttle: true
auto_takeoff_land:
  enable: true
  enable_auto_arm: true
  no_RC: true
  takeoff_height: 1
  takeoff_land_speed: 0.3
thrust_model:
  print_value: false
  accurate_thrust_model: false
  K1: 0.7583
  K2: 1.6942
  K3: 0.6786
  hover_percentage: 0.72
gain:
  normal:
    Kp0: 3.5
    Kp1: 3.5
    Kp2: 3.5
    Kv0: 3.5
    Kv1: 3.5
    Kv2: 3.5
    Kvi0: 0
    Kvi1: 0
    Kvi2: 0
    Kvd0: 0
    Kvd1: 0
    Kvd2: 0
    KAngR: 10
    KAngP: 10
    KAngY: 5
  track:
    Kp0: 1.5
    Kp1: 1.5
    Kp2: 1.5
    Kv0: 1.5
    Kv1: 1.5
    Kv2: 1.5
    Kvi0: 0.2
    Kvi1: 0.2
    Kvi2: 0.2
    Ka0: 1
    Ka1: 1
    Ka2: 1
    Kyaw: 10
    Krp: 30
rotor_drag:
  x: 0
  y: 0
  z: 0
  k_thrust_horz: 0
msg_timeout:
  odom: 0.5
  rc: 0.5
  cmd: 0.5
  imu: 0.5
  bat: 0.5
